setup.py

```diff
@@ -37,7 +37,7 @@
         # TODO: fix dependencies when we will publish the legion packages to pypi repo
         # TODO: consider Pipenv usage
         'legion-sdk @ git+https://github.com/legion-platform/legion.git@1.0.0-rc18#egg=legion-sdk&subdirectory=legion/sdk',
-        'mlflow==1.0.0',
+        'mlflow==1.3.0',
```

Comment:
Is it okay that we hardcode the version?
`mlflow<=1.3.0` ?

Reply:
Yes, agree, looks better

Reply:
@mcdoker18 updated